docs/docs/dev/deployments/multi-zone.md

```diff
@@ -1,13 +1,127 @@
+
+The DNS resolver is embedded in each data-plane proxy and configured through XDS. It resolves each service address to a virtual IP address for all service-to-service communication.
+
+The global control plane and the zone control planes communicate to synchronize resources such as Kuma policy configurations over Kuma Discovery Service (KDS), which is a protocol based on xDS.
+
+:::tip
+A zone ingress is not an API gateway. Instead, it is specific to internal cross-zone communication within the mesh. API gateways are supported in Kuma [gateway mode](../documentation/dps-and-data-model.md) which can be deployed in addition to zone ingresses.
+:::
+
+### Components of a multi-zone deployment
+
+A multi-zone deployment includes:
+
+* The **global control plane**:
+  * Accept connections only from zone control planes.
+  * Accept creation and changes to [policies](/policies) that will be applied to the data plane proxies.
+  * Send policies down to remote control-planes.
+  * Send zone ingresses down to remote control-plane.
+  * Keep an inventory of all dataplanes running in all zones (this is only done for observability but is not required for operations).
+  * Reject connections from data-plane proxies.
+* The **zone control planes**: 
+  * Accept connections from data plane proxies started within this zone.
+  * Receive policy updates from the global control plane.
+  * Send data-plane and zone ingress changes to the global control plane.
+  * Compute and send configurations using XDS to the local data-plane proxies.
+  * Update list of services who exist in the zone in the zone ingress.
+  * Reject policy changes that do not come from global.
+* The **data-plane proxies**:
+  * Connect to the local zone control plane.
+  * Receive configurations using XDS from the local zone control plane.
+  * Connect to other local data plane proxies.
+  * Connect to remote zone ingresses for sending cross zone traffic.
+  * Receive traffic from local data-plane proxies.
+* The **zone ingress**:
+  * Receive XDS configuration from the local zone control plane.
+  * Proxy traffic from remote data plane proxies to local data plane proxies.
+
+## Limitations
+
+It is not possible to route cross-zone traffic on a subset of dataplanes with the same `kuma.io/service`.
+
+This means that complex [Traffic-routes](../policies/traffic-route.md) or [Virtual-outbound](../policies/virtual-outbound.md) will not route any traffic across zones.
+
+## Failure modes
+
+### Global control-plane offline
+
+* Policy updates will be impossible 
+* Change in service list between zones will not propagate:
+  * New services will not be discoverable in other zones
+  * Services removed from a zone will still appear available in other zones
+* You won't be able to disable or delete a zone
+
+::: tip
+Note that both local and cross-zone application traffic is not impacted by this failure case.
```

Comment:
also add that you can bring up and down instances in the local zone and this information will be propagated in this zone